.eslintignore

```diff
@@ -0,0 +1 @@
+src/serviceWorker.ts
```

Comment:
Do you want to snatch gitignore config from youth-ui or kukku ? Or in separate PR?

Reply:
I guess it's the same that CRA generates.

Reply:
Sorry, wrong file, thought this is gitignore